network/network-tutorial-standalone.md

```diff
@@ -306,7 +306,7 @@ connected to both networks.
 
     $ docker run -dit --name alpine4 --network alpine-net alpine ash
 
-    $ docker network connect bridge alpine4
+    $ docker network connect alpine-net alpine3
```

Comment:
Looking at the description above, the original text was correct;

> you need to use `docker network connect` afterward to connect `alpine4` to the `bridge` network as well.

The example describes that container `alpine4` is connected to network `alpine-net`, and after that is connected to network `bridge` (so that it is now connected to both networks)